components/Profile/index.js

```diff
@@ -66,52 +72,81 @@ export default function Profile({ userProfile }) {
     const newEmail = event.target.value;
     setEmail(newEmail);
   }
-  /* 
-  function handleAvatarChange(event) {
-    event.preventDefault();
-    const newAvatar = event.target.files[0];
-    setAvatar(newAvatar);
-  } */
+  async function handleAvatarChange(imageURL) {
+    try {
+      const requestBody = { imageURL: imageURL };
+      console.log(requestBody.imageURL);
+      const response = await fetch("/api/images/imagesChange", {
+        method: "PUT",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(requestBody),
+      });
+
+      if (response.ok) {
+        setAvatar(imageURL);
+        mutate();
+        console.log("Avatar updated in MongoDB:", imageURL);
+      } else {
+        console.error(
+          "Unfortunately failed to update avatar in MongoDB",
+          error
+        );
+      }
+    } catch (error) {
+      console.error("Failed to update avatar in MongoDB", error);
```

Comment:
See comments above ⏫

Reply:
I would leave it, cause it is an "important" log in the catch-block :)